src/applications/facility-locator/constants/mock-facility-data.json.org

```diff
@@ -0,0 +1,1031 @@
+{
```

Comment:
Is the `.org` extension intentional?

Reply:
@cvalarida thanks for approving this, the file is used to testing. I use it at the moment for mock data. Will remove later.